api/v1alpha3/virtualmachineimage_types.go

```diff
@@ -200,6 +215,11 @@ type VirtualMachineImageStatus struct {
 
 	// +optional
 
+	// DiskInfo describes the observed disk information for this image.
```

Comment:
Nit: maybe we want to make this description & field name plural?

Reply:
I think it is fine. DiskInfo == DiskInformation, and "information" can mean multiple disks IMO.

Reply:
I do like `Status.Disks` but totally cool with whatever y'all want to change it to or keep it as-is.